OMCompiler/Compiler/Script/CevalScriptBackend.mo

```diff
@@ -3387,6 +3391,161 @@ algorithm
   end matchcontinue;
 end runFrontEndWork;
 
+
+/* -------------------------------------------------------------------
+                         RUN NEW FRONTEND
+   ------------------------------------------------------------------- */
+public function runNewFrontEnd
+  "author: kabdelhak 2020-03
+   Only run the new frontend pipeline and return the flat model instead of
+   DAE representation. Used for new backend and should replace runFrontEnd
+   at some point."
+  input FCore.Cache cache;
+  input FCore.Graph env;
+  input Absyn.Path className;
+  output Option<FlatModel> optFlatModel = NONE();
+  output Option<FunctionTree> optFuncs = NONE();
+  input Boolean relaxedFrontEnd "Do not check for illegal simulation models, so we allow instantation of packages, etc";
+protected
+  FlatModel flatModel;
+  FunctionTree funcs;
+  Boolean b;
```

Comment:
Change to programIsLoaded? or something more descriptive

Reply:
Sure, i just copied some other work here. But i am all for more descriptive names.